src/main/java/it/gov/pagopa/pu/organization/config/RepositoryRestCustomConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package it.gov.pagopa.pu.organization.config;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.metamodel.EntityType;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.rest.core.config.RepositoryRestConfiguration;
+import org.springframework.data.rest.webmvc.config.RepositoryRestConfigurer;
+
+@Configuration
+public class RepositoryRestCustomConfiguration {
+
+  private EntityManager entityManager;
+  public RepositoryRestCustomConfiguration(EntityManager entityManager){
+    this.entityManager = entityManager;
+  }
+  @Bean
+  public RepositoryRestConfigurer repositoryRestConfigurer() {
+    return RepositoryRestConfigurer.withConfig( config() );
+  }
+
+  Consumer<RepositoryRestConfiguration> config () {
```

Comment:
I would let the lambda inside the method invocation instead of moving it inside a separate method...
Lambdas have been created to avoid the creation of class having just one method, to create a method to return a lambda, if not entirely reused, could be less readable... It could be more readable to have a method which represent the lambda body

eg:
```
private void configureRepositoryRest(RepositoryRestConfiguration config){
...
}
```

But the @Bean method has just one instruction whose purpose is to call the lambda, so the lambda represent its main focus... So I would keep it simple and avoid to create a new method (the main method has not other purpose and the lambda is not more used outside)

Reply:
updated